Homework_Basic_Python.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assignment 1\n",
+    "\n",
+    "Calculate the sum of all the squared numbers from 1 to 100. Keep in mind that the `range()` function excludes the given upper limit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "385\n"
+     ]
+    }
+   ],
+   "source": [
+    "uno = [d**2 for d in range(1,11)]\n",
```

Comment:
this is very "pythonic" (how python programmer do it) but do you understand what's happening here?